cloudify_docker_image/make_cfy_image.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -eu
+if [ $# -eq 0 ]
+  then
+    echo "No arguments supplied. Please supply the CFY RPM URL as the first argument."
+    exit
+fi
+# RPM URL must be supplied as the first argument
+CFY_RPM_URL=$1
+CFY_RPM="cfy.rpm"
+CONTAINER_NAME="cfy_manager"
+IMAGE_PUB_NAME="docker_cfy_manager"
+
+echo "Building base docker image..."
+BASE_IMAGE_ID=$(docker build . -q)
+echo "Built base image ${BASE_IMAGE_ID}"
+docker run --name $CONTAINER_NAME -d -v /sys/fs/cgroup:/sys/fs/cgroup:ro --tmpfs /run \
+	--tmpfs /run/lock --security-opt seccomp:unconfined --cap-add SYS_ADMIN $BASE_IMAGE_ID
+CONTAINER_IP=$(docker inspect --format='{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' ${CONTAINER_NAME})
+
+echo "Installing cfy..."
+docker exec -it $CONTAINER_NAME sh -c "curl $CFY_RPM_URL -o ~/$CFY_RPM && rpm -i ~/$CFY_RPM && rm -f ~/$CFY_RPM"
+echo "Creating install config file..."
+echo "manager:
+  private_ip: ${CONTAINER_IP}
+  public_ip: ${CONTAINER_IP}
+  set_manager_ip_on_boot: true
+  security:
+    admin_password: admin" > config.yaml
+
+docker cp config.yaml ${CONTAINER_NAME}:/opt/cloudify-manager-install/config.yaml
+
+echo "Starting up sshd service..."
+docker exec -d ${CONTAINER_NAME} sh -c "systemctl start sshd"
```

Comment:
Can we add this line to the Dockerfile? I don't know if that's possible, but that would be awesome.

Reply:
I tried `systemctl enable sshd.service; systemctl start sshd` but it won't work. I think we should leave it as an enhancement issue